services/scabbard/libscabbard/src/migrations/diesel/postgres/migrations/2022-03-29-160500-create-scabbard-store-tables/down.sql

```diff
@@ -13,16 +13,16 @@
 -- limitations under the License.
 -- -----------------------------------------------------------------------------
 
-DROP TABLE IF EXISTS consensus_coordinator_context;
```

Comment:
the two_pc naming scheme is a bit odd.  Perhaps these tables should be be prefixed with`consensus_2pc_`.

This naming convention then would continue with additional consensus algorithms:

* Three-Phase Commit: `consensus_3pc_`
* PBFT: `consensus_pbft_`

and so on.